1/SumOfEvenNumbers.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AgileFreaks_test1
+{
+    public class SumOfEvenNumbers
+    {
+        private int s;
+
+        public SumOfEvenNumbers()
+        {
+            s = 0;
```

Comment:
If you indeed want that sum to be a member on an instance, make the array a hard dependency for the instance.